app/src/main/res/layout/saved_photos_fragment.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler_view"
```

Comment:
El nombre de la vista debería ser único en el proyecto. Yo lo llamaría algo así como saved_photos_recycler

Reply:
Mmm dentro de este binding concreto, sólo hay un `RecyclerView`. 

Encuentro el beneficio a alargar los nombres cuando nos van a aportar algo que no sepamos en el contexto en cuestión. Pero en este caso, tanto en el Layout como en el Fragment luego, no nos aportaría mucho el mencionar `binding.savedPhotosRecycler` en vez de `binding.recycler` o `binding.recyclerView.

Ya estamos en el fragment `SavedPhotosFragment`. Claramente el Recycler es para este Fragment.

Antonio en sus ejemplos no veo que use esta nomenclatura: https://androidboosttraining.club.hotmart.com/lesson/meLq60Ej7n/evento-de-click-en-el-adapter

Pero bueno, es mi opinión. Si estáis de acuerdo los demás en utilizar esta nomenclatura, adelante.